examples/dmm/polyphonic_data_loader.py

```diff
@@ -117,10 +117,10 @@ def get_mini_batch(mini_batch_indices, sequences, seq_lengths, cuda=False):
     # get mask for mini-batch
     mini_batch_mask = get_mini_batch_mask(mini_batch, sorted_seq_lengths)
 
-    # wrap in PyTorch Tensors
-    mini_batch = torch.tensor(mini_batch)
-    mini_batch_reversed = torch.tensor(mini_batch_reversed)
-    mini_batch_mask = torch.tensor(mini_batch_mask)
+    # wrap in PyTorch Tensors, using default tensor type
+    mini_batch = torch.tensor(mini_batch).type(torch.empty(1).type())
```

Comment:
why is this needed this seems crazy?

Reply:
For the reason, mentioned above. Any numpy array when wrapped into a torch tensor is converted to a double tensor by default. This recasts it based on the default tensor type. Let me know if there is a less uglyish way to do this.

Reply:
type_as would marginally decrease verbosity/ugliness